### PR TITLE
Improves New-StoragePool

### PR DESCRIPTION
**Pull request by exchange12rocks**
1. Improves formatting in examples.
2. Some parameters in reality have default values, but the documentation misses it. Not anymore.

### CODE REVIEW

**Comment by e0i** [comment]
CC @JohanFreelancer9

**Review by get-itips — APPROVED** [review]

File: docset/windows/storage/New-StoragePool.md [hunk 1]
```diff
@@ -72,13 +72,14 @@ The **New-StoragePool** cmdlet creates a new storage pool using a group of physi
...
 ### Example 1: Create a new storage pool using Storage Spaces
 ```
-This line uses the **Get-PhysicalDisk** cmdlet to get all PhysicalDisk objects than are not yet in a (concrete) storage pool, and assigns the array of objects to the $PhysicalDisks variable.
 PS C:\> $PhysicalDisks = (Get-PhysicalDisk -CanPool $True)
-
-This line creates a new storage pool using the $PhysicalDisks variable to specify the disks to include from the WindowsStorage subsystem (specified with a wildcard * to remove the need to modify the friendly name for different computers).
 PS C:\> New-StoragePool -FriendlyName CompanyData -StorageSubsystemFriendlyName "Windows Storage*" -PhysicalDisks $PhysicalDisks
 ```
 
+The first line uses the **Get-PhysicalDisk** cmdlet to get all PhysicalDisk objects than are not yet in a (concrete) storage pool, and assigns the array of objects to the $PhysicalDisks variable.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/New-StoragePool.md:79]:
```suggestion
The first line uses the **Get-PhysicalDisk** cmdlet to get all PhysicalDisk objects that are not yet in a (concrete) storage pool and assigns the array of objects to the $PhysicalDisks variable.
```

File: docset/windows/storage/New-StoragePool.md [hunk 2]
```diff
@@ -90,10 +91,22 @@ PS C:\> New-StoragePool -FriendlyName CompanyData -StorageSubsystemFriendlyName
 This example creates a new storage pool named CompanyData using the Windows Storage subsystem and sets default values for virtual disk creation.
 
 ### Example 3: Create a new storage pool, virtual disk, partition, and volume
+
+The first line (`$PhysicalDisks = …`) gets the storage subsystem object for the Windows Storage subsystem, passes it to the **Get-PhysicalDisk** cmdlet, which then gets the physical disks in the specified subsystem that are available to add to a storage pool, and assigns these disks to the $PhysicalDisks variable.
+
+The second line of the command has five parts, connected by the pipeline (|).
+
+The first part (`New-StoragePool …`) creates a new storage pool using the physical disks in the $PhysicalDisks variable, and then passes the new storage pool down the pipeline. All of the following commands are logically part of one command and should be entered as such.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/New-StoragePool.md:99]:
```suggestion
The first part (`New-StoragePool …`) creates a new storage pool using the physical disks in the $PhysicalDisks variable and then passes the new storage pool down the pipeline. All of the following commands are logically part of one command and should be entered as such.
```

File: docset/windows/storage/New-StoragePool.md [hunk 3]
```diff
@@ -90,10 +91,22 @@ PS C:\> New-StoragePool -FriendlyName CompanyData -StorageSubsystemFriendlyName
...
+
+The second part (`New-VirtualDisk …`) creates a new virtual disk on the passed in storage pool and then passes the new virtual disk down the pipeline.
+The third part (`Initialize-Disk …`) initializes the disk that was passed in, and then passes the disk down the pipeline.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/New-StoragePool.md:102]:
```suggestion
The third part (`Initialize-Disk …`) initializes the disk that was passed in and then passes the disk down the pipeline.
```

File: docset/windows/storage/New-StoragePool.md [hunk 4]
```diff
@@ -298,13 +304,14 @@ Accepted values: Unknown, Thin, Fixed
...
-Default value: None
+Default value: Fixed
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -ResiliencySettingNameDefault
-Specifies the default resiliency setting (also known as storage layout) to use for virtual disks created in the specified storage pool. The supported resiliency settings vary by storage subsystem. Acceptable values for the Windows Storage subsystem are Mirror, Parity, and Simple.
+Specifies the default resiliency setting (also known as storage layout) to use for virtual disks created in the specified storage pool. The supported resiliency settings vary by storage subsystem.
+For the Windows Storage subsystem acceptable values are Mirror, Parity, and Simple. "Mirror" is the default value.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/New-StoragePool.md:314]:
```suggestion
For the Windows Storage subsystem, acceptable values are Mirror, Parity, and Simple. "Mirror" is the default value.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: docset/windows/storage/New-StoragePool.md [hunk 5]
```diff
@@ -72,13 +72,14 @@ The **New-StoragePool** cmdlet creates a new storage pool using a group of physi
...
-This line uses the **Get-PhysicalDisk** cmdlet to get all PhysicalDisk objects than are not yet in a (concrete) storage pool, and assigns the array of objects to the $PhysicalDisks variable.
 PS C:\> $PhysicalDisks = (Get-PhysicalDisk -CanPool $True)
-
-This line creates a new storage pool using the $PhysicalDisks variable to specify the disks to include from the WindowsStorage subsystem (specified with a wildcard * to remove the need to modify the friendly name for different computers).
 PS C:\> New-StoragePool -FriendlyName CompanyData -StorageSubsystemFriendlyName "Windows Storage*" -PhysicalDisks $PhysicalDisks
 ```
 
+The first line uses the **Get-PhysicalDisk** cmdlet to get all PhysicalDisk objects that are not yet in a (concrete) storage pool and assigns the array of objects to the $PhysicalDisks variable.
+
+The second line creates a new storage pool using the $PhysicalDisks variable to specify the disks to include from the WindowsStorage subsystem (specified with a wildcard * to remove the need to modify the friendly name for different computers).
```
Comment (tseward) [inline docset/windows/storage/New-StoragePool.md:81]: Please back tick `$PhysicalDisks`.

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by dariomws — APPROVED** [review]
I think you can also leave the acceptable values here.

**Review by JohanFreelancer9 — APPROVED** [review]
Approved with @tseward's and @dariomws's suggestions.

**Comment by exchange12rocks** [comment]
I reverted back the acceptable values changes, so as I understand, this PR is ready for merging now.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**